.travis.yml

```diff
@@ -5,39 +5,31 @@ branches:
 cache:
   ccache: true
   directories:
-   - $DEPOT_TOOLS_PATH
-   - $BUILD_PATH
+   - $DENO_BUILD_PATH
 env:
   global:
-    - BUILD_PATH=$HOME/out/Default
-    - DEPOT_TOOLS_PATH=$HOME/depot_tools
-before_install: |
-  if ! [ -x $DEPOT_TOOLS_PATH/gclient ]; then
-    rm -rf $DEPOT_TOOLS_PATH
-    git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH
-  fi
+    # is_debug, use_allocator, and ccache are used to speed travis.
+    # use_custom_libcxx=false and use_sysroot=false seem to be required to build on
+    # Ubuntu 14.04
+    # Help: How do you wrap long lines here?
```

Comment:
https://stackoverflow.com/a/21699210

Reply:
I'll just leave the comment - still haven't figure it out and it's not worth our energy to figure it out.